wiki/People/The_Team/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -0,0 +1,83 @@
+
+If it is found the BN is indeed not meeting activity, they will be either warned or removed depending on severity. Beatmap Nominators who were removed for activity in this way may apply again in 30 days, provided they have at least 8 mods to show they have come back to being active as a modder again.
+
+### Resignations
+
+Beatmap Nominators can resign from the group at any time via the NAT website. When this happens, three NAT members will be assigned to evaluate the nominator one last time, in order to determine if the resignation is on "good" terms or on "standard" terms. This evaluation is given a 24-hour deadline and announced to the NAT members in their respective Discord channel. Beatmap Nominators who resign on good terms may be able to fast-track back to being a full BN if they choose to apply again within 1 year of resignation. Those who resign on standard terms will have to start from probation if they pass their reapplication. 
+
+Generally, good- and standard-term evaluations are relatively simple. Beatmap Nominators who have not been recently warned or placed on probation, and were not about to be warned for issues, will usually resign on good terms. Standard terms is typically used for nominators who have been warned recently, are on probation, or were about to be warned for issues on their upcoming current BN evaluation.
+
+Once a consensus for the resignation terms has been reached, the NAT will send the BN a message via the [mappersguild bot](https://osu.ppy.sh/users/23648635) informing them of their status and when they can reapply to become a BN if they so choose.
+
+### Mock evaluations
+
+In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. While NAT members are obliged to participate in the evaluation process, there are no consequences for nominators who do not fulfill the given assignment. For the mock evaluation they participated in, the nominator will be able to read the individual evaluations in an anonymised format once the application is archived.
+
+Mock evaluations allow the NAT to scout for future NAT members, as well as for Beatmap Nominators to see how the NAT evaluates applications and learn from them if joining the NAT in the future is something they are interested in.
```

Comment:
```suggestion
In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. For mock evaluations they participated in, nominators will be able to read the individual evaluations in an anonymised format once the application is archived.

Mock evaluations allow the NAT to scout for future NAT members, as well as for Beatmap Nominators to see how the NAT evaluates applications and learn from them if joining the NAT in the future is something they are interested in. As such, there are no consequences for not completing them.
```